cmd/cvc-operator/controller/cstorvolumeclaim.go

```diff
@@ -521,6 +529,92 @@ func randomizePoolList(list *apis.CStorPoolInstanceList) *apis.CStorPoolInstance
 	for _, randomIdx := range perm {
 		res.Items = append(res.Items, list.Items[randomIdx])
 	}
-
 	return res
 }
+
+// getOrCreatePodDisruptionBudget will does following things
+// 1. It tries to get the PDB that was created among volume replica pools.
+// 2. If PDB exist it returns the PDB.
+// 3. If PDB doesn't exist it creates new PDB(With CSPC hash)
+func getOrCreatePodDisruptionBudget(
+	cvObj *apis.CStorVolume, cspcName string) (*policy.PodDisruptionBudget, error) {
+	poolNames, err := cvr.GetReplicaPoolNames(cvObj)
+	if err != nil {
+		return nil, errors.Wrapf(err,
+			"failed to get volume replica pool names of volume %s",
+			cvObj.Name)
+	}
+	pdbLabels := cvclaim.GetPDBPoolLabels(poolNames)
+	labelSelector := apispdb.GetPDBLabelSelector(pdbLabels)
+	pdbList, err := apispdb.KubeClient().
+		WithNamespace(getNamespace()).
+		List(metav1.ListOptions{LabelSelector: labelSelector})
+	if err != nil {
+		return nil, errors.Wrapf(err,
+			"failed to list PDB belongs to pools %v", pdbLabels)
+	}
+	if len(pdbList.Items) > 1 {
```

Comment:
does this work even if multiple CVC controllers are running at same time?

Reply:
Looks like we have to take a lease on PDB also when we take on CVC.

Reply:
It will work without any changes